components/busy-ui/busy-ui.js

```diff
@@ -14,6 +14,14 @@
  * <busy-ui data-message="hello world"></busy-ui>
  */
 export class BusyUi extends HTMLElement {
+    /**
+     * @method observedAttributes - the attributes to watch for changes
+     * @return {[string]}
+     */
+    static get observedAttributes() {
+        return ['data-progress'];
```

Comment:
you also need to have data-message in here otherwise it wont trigger

Reply:
data-message added.